cleanlab/count.py

```diff
@@ -263,16 +263,15 @@ def _compute_confident_joint_multi_label(
 
     # Compute unique number of classes K by flattening labels (list of lists)
     K = len(np.unique([i for lst in labels for i in lst]))
```

Comment:
If we're computing num_classes based on unique values, the docstring should explicitly state that all classes must be present in labels to avoid possible user confusion like we have seen in some Issues.

Reply:
I second this comment ^^. This is so true in multi-label and you are likely to run into this case give you explicitly don't ask for one hot encoded label for multi-label settings. 
I more I think about it, the more I feel, that it will be better to have a different abstraction for multi-label than trying to wedge it in via flag. At the same time, I appreciate that this support is experimental so all good! 

Left some thoughts here https://github.com/cleanlab/cleanlab/issues/263#issuecomment-1133885728 but I won't say my thoughts are well thought through just yet.

Reply:
clarified this in labels param description. Also the following tutorials PR helped address this: https://github.com/cleanlab/cleanlab/pull/279